example/babel.config.js

```diff
@@ -3,5 +3,6 @@ module.exports = {
   plugins: [
     '@babel/plugin-proposal-export-namespace-from',
     'react-native-reanimated/plugin',
```

Comment:
[react-native-reanimated/plugin](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started/#step-2-add-reanimateds-babel-plugin) has to be listed last.

Reply:
fixed, thanks